db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/QueryTreeNode.java

```diff
@@ -1278,6 +1278,15 @@ public final LanguageConnectionContext getLanguageConnectionContext(){
         return lcc;
     }
 
+    /**
+     * True if this predicate is used in a trigger query.
+     */
+    public final boolean compilingTrigger() {
+        LanguageConnectionContext lcc = getLanguageConnectionContext();
+        boolean compilingTrigger = lcc.compilingTrigger();
+        return compilingTrigger;
+    }
+
```

Comment:
as far as i saw it, this is only used if we are just compiling a trigger, right? after that is done, compilingTrigger is set to false again.
(nitpic) that's a quite long way to write `return getLanguageConnectionContext().compilingTrigger();` :-P

Reply:
Right.  Simplified the code as suggested.